src/db_stats.cpp

```diff
@@ -52,22 +52,24 @@ class StatsCollector
         std::lock_guard lock(stats_mutex_);
         stats_.pending_duration_ += std::chrono::duration_cast<metricq::Duration>(pending_duration);
         stats_.pending_count_--;
+        stats_.started_count_++;
         stats_.active_count_++;
     }
 
     template <typename T>
     void complete(T active_duration, size_t data_size)
     {
         std::lock_guard lock(stats_mutex_);
-        stats_.ready_count_++;
+        stats_.completed_count_++;
         stats_.active_count_--;
         stats_.active_duration_ += std::chrono::duration_cast<metricq::Duration>(active_duration);
         stats_.data_size_ += data_size;
     }
 
     struct Stats
     {
-        size_t ready_count_ = 0;
+        size_t completed_count_ = 0;
+        size_t started_count_ = 0;
```

Comment:
It would be nice if we can distinguish between the counts that are only incremented and reset and those that are incremented and decremented as we go. In the hope that someone is annoyed and comes up with a better name, I'll suggest `count_since_reset` and `current_number`.

P.S. I'm aware of the irony that I started this mess and am now complaining about it.